### PR TITLE
Add mkt statement and link to RHACM SOLP

### PR DESCRIPTION
**Pull request by cadawson**
A request came from Debbie Margulies from marketing through Michelle Bearer to add this statement and link. Thanks!

### CODE REVIEW

File: migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc [hunk 1]
```diff
@@ -12,6 +12,8 @@ The most effective way to migrate from {product-title} 3 to 4 is by using a CI/C
 
 If you do not have a CI/CD pipeline or if you are migrating stateful applications, you can use the {mtc-full} ({mtc-short}) to migrate your application workloads.
 
+You can use Red Hat Advanced Cluster Management for Kubernetes to help you import and manage your {product-title} 3.x clusters easily, enforce policies, and redeploy your applications. Take advantage of the link:https://www.redhat.com/en/engage/free-access-redhat-e-202202170127[free subscription] to use Red Hat Advanced Cluster Management to simplify your migration process.
```
Comment (bmcelvee) [inline migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc:15]: Suggestion [optional]: 
s/3.x/3

Comment (cadawson) [reply migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc:15]: Changed in updated PR#44060. Thanks!

**Comment by cadawson** [comment]
Closing this, as it was updated in the main branch as recommended. Updated PR#44060.

File: migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc (hunk 1 above)
Comment (cadawson) [reply migrating_from_ocp_3_to_4/about-migrating-from-3-to-4.adoc:15]: Changed in updated PR#44060. Thanks!

**Pull request closed without merging**